src/test/scala/com/microsoft/hyperspace/index/IndexManagerTests.scala

```diff
@@ -264,10 +268,23 @@ class IndexManagerTests extends HyperspaceSuite with SQLHelper {
           .parquet(testPath)
         val df = spark.read.parquet(testPath)
         hyperspace.createIndex(df, indexConfig)
-        val ex = intercept[HyperspaceException] {
-          hyperspace.refreshIndex(indexConfig.indexName)
+        val indexPath = PathUtils.makeAbsolute(s"$systemPath/${indexConfig.indexName}")
+        val logManager = IndexLogManagerFactoryImpl.create(indexPath)
+        val latestId = logManager.getLatestId().get
+
+        hyperspace.refreshIndex(indexConfig.indexName)
+        // Check that no new log files were created in this operation.
+        assert(latestId == logManager.getLatestId().get)
+
+        // Check emitted events.
+        MockEventLogger.emittedEvents match {
+          case Seq(
+              _: CreateActionEvent,
+              _: CreateActionEvent,
+              _: RefreshAppendActionEvent,
+              _: RefreshAppendActionEvent) => // pass
+          case _ => fail()
```

Comment:
Can you reset before calling `refreshIndex` and check only one event is emitted (also check the message).

Reply:
Thanks @imback82, sure will do. there will still be 2 events: RefreshAppendActionEvent("Operation started") and ...("Operation Ended").

Reply:
thanks @imback82 , updated